locations/spiders/sheetz.py

```diff
@@ -1,8 +1,39 @@
+from enum import Enum
+
 import scrapy
 
+from locations.categories import Categories, Extras, Fuel, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
 from locations.items import Feature
 from locations.user_agents import BROWSER_DEFAULT
 
+FEATURES_MAPPING = {
+    "alcohol": None,
+    "atm": Extras.ATM,
+    "bulkDef": None,
+    "carWash": Extras.CAR_WASH,
+    "delivery": Extras.DELIVERY,
+    "driveThru": Extras.DRIVE_THROUGH,
+    "diesel": Fuel.DIESEL,
+    "e0": Fuel.ETHANOL_FREE,
+    "e15": Fuel.E15,
+    "e85": Fuel.E85,
+    "freeWifi": Extras.WIFI,
+    # TODO: map high flow auto diesel
+    "highFlowAutoDiesel": None,
+    "highFlowDiesel": Fuel.HGV_DIESEL,
+    "kerosene": Fuel.KEROSENE,
+    "propane": Fuel.PROPANE,
+    "pncAtm": Extras.ATM,
+    "showers": Extras.SHOWERS,
+    "truckDieselLanes": None,
+    # TODO: availability of truck scales seems important for trucks
+    "truckScales": None,
```

Comment:
`amenity=weighbridge` is the relevant OSM tag to use here.

https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dweighbridge

Reply:
This is the only other improvement I can see. Looks pretty good otherwise. Lots of detailed tagging with this brand :)

Reply:
Thanks @davidhicks! 
For now I've added a TODO for them. Similar to `charging_station` we probably want to output it as a separate item at some point.